datadog_checks_base/datadog_checks/checks/prometheus/base_check.py

```diff
@@ -7,7 +7,13 @@
 from .. import AgentCheck
 from ...errors import CheckException
 
+
 class Scraper(PrometheusScraper):
```

Comment:
The name of this class is admittedly bad and I'd take any suggestions for renaming. Maybe rename the mixin to `PrometheusScraperMixin` and also have `PrometheusScraper` as the standalone version?

Reply:
Your suggestion sounds good to me, you'll need to update a few things (export, Istio check) for renaming.

I'd say it'd be nice to have but as it's in the `prometheus` module it's not shocking to be named just `Scraper`.